build-docs-rsync

```diff
@@ -6,17 +6,33 @@ GIT_RESET="git reset --quiet --hard origin/master"
 cd /local/repos/phpdoc/phd
 $GIT_FETCH && $GIT_RESET >/tmp/log-phd 2>&1
 
-# Update docs sources
-cd /local/repos/phpdoc
-svn up >/tmp/log-svn 2>&1
+BASE_DIR=/local/repos/phpdoc-git
+
+assertUpToDateRepo() {
+    if [ ! -d $BASE_DIR/$1 ]; then
+        git clone https://git.php.net/repository/doc/$2.git $BASE_DIR/$1
+    fi
+    git -C $BASE_DIR/$1 fetch origin && git -C $BASE_DIR/$1 reset --hard origin/master
+}
```

Comment:
The old `svn up` output used to get logged to `/tmp/log-svn`.  No logging happens for these git commands; I suggest logging the output somewhere (e.g. `/tmp/log-git`).